src/pages/publication/public_publication/PublicPublicationContent.tsx

```diff
@@ -0,0 +1,210 @@
+import NotFound from '../../errorpages/NotFound';
+import Card from '../../../components/Card';
+import Heading from '../../../components/Heading';
+import LabelContentRow from '../../../components/LabelContentRow';
+import Label from '../../../components/Label';
+import NormalText from '../../../components/NormalText';
+import { licenses } from '../../../types/file.types';
+import { getNpiDiscipline } from '../../../utils/npiDisciplines';
+import { StyledNormalTextPreWrapped } from '../../../components/styled/Wrappers';
+import { displayDate } from '../../../utils/date-helpers';
+
+const StyledContentWrapper = styled.div`
+  display: flex;
+  padding-top: 1rem;
+
+  @media (max-width: ${({ theme }) => theme.breakpoints.values.md + 'px'}) {
+    flex-direction: column;
+  }
+`;
+
+const StyledSidebar = styled.div`
+  min-width: 15rem;
+  padding: 1rem 0;
+`;
+
+const StyledMainContent = styled.div`
+  flex: 1;
+  padding: 1rem 2rem;
+`;
+
+const StyledSidebarCard = styled(Card)`
+  padding: 1rem 0.5rem;
+`;
+
+const StyledLicenseCard = styled(Card)`
+  display: grid;
+  grid-template-areas: 'image label' 'image description';
+  column-gap: 1rem;
+  align-items: center;
+  justify-content: left;
+  margin: 0.5rem 0;
+  padding: 1rem;
+`;
+
+const StyledLicenseImage = styled.img`
+  grid-area: image;
+`;
+
+const StyledLicenseLabel = styled(Label)`
+  grid-area: label;
+`;
+
+const StyledNormalText = styled(StyledNormalTextPreWrapped)`
+  grid-area: description;
+`;
+
+const StyledTextContainer = styled.div`
+  display: inline-block;
+  margin: 1rem 0;
+`;
+
+const StyledLabel = styled(Label)`
+  display: inline-block;
+`;
+
+const StyledTextDescription = styled(NormalText)`
+  display: inline;
+  margin-left: 1rem;
+`;
+
+const StyledTag = styled.div`
+  display: inline;
+  margin-right: 1rem;
+`;
+
+interface PublicPublicationContentProps {
+  publication: Publication;
+}
+
+const PublicPublicationContent: FC<PublicPublicationContentProps> = ({ publication }) => {
+  const { t } = useTranslation('publication');
+
+  const {
+    abstract,
+    contributors,
+    date,
+    description,
+    mainTitle,
+    npiSubjectHeading,
+    reference: { doi, publicationContext, publicationInstance },
+    series,
+    tags,
+  } = publication ? publication.entityDescription : emptyPublication.entityDescription;
+
+  // Show only the license for the first file for now
+  const currentLicense = publication?.fileSet?.files[0]?.license ?? null;
+  const selectedLicense = licenses.find((license) => license.identifier === currentLicense?.identifier);
+
+  return (
+    <ContentPage>
+      {publication ? (
+        <>
+          <Heading>{mainTitle}</Heading>
+          {contributors && <PublicPublicationAuthors contributors={contributors} />}
+          <StyledContentWrapper>
+            <StyledSidebar>
+              {/* <StyledSidebarCard>
+                    TODO: Put affiliations to contributors here
```

Comment:
kommentaren her skulle vært borte for lenge siden

Reply:
![Skjermbilde 2020-06-11 kl  12 49 23](https://user-images.githubusercontent.com/6595982/84376853-01d64680-abe2-11ea-9ab5-399b2b7691ef.png)

Mangler vi ikke denne boksen da?

Reply:
ser nå at vi har tekst, men ikke boks med link. men kanskje ikke så farlig?

Reply:
tror det bare var en todo for å vise affiliations generelt. aner ikke hva det eventuelt skulle blitt linket til heller